Clean up some type annotations in torch/testing/_internal

r-barnes opened the pull request:
Summary: Upgrades type annotations from Python2 to Python3

Test Plan: Sandcastle tests

Differential Revision: D25717560

pritamdamania87 reviewed (approved):
LGTM, please make sure all circleci jobs succeed before landing :)

r-barnes committed:
Clean up some type annotations in torch/testing/_internal (#50078)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/50078

Upgrades type annotations from Python2 to Python3

Test Plan: Sandcastle tests

Reviewed By: pritamdamania87

Differential Revision: D25717560

fbshipit-source-id: f77f055d777c3793ca5ef0ab2dd8126720f6afa6
